test/e2e/testnet/node.go

```diff
@@ -415,3 +416,18 @@ func (n *Node) Upgrade(version string) error {
 func DockerImageName(version string) string {
 	return fmt.Sprintf("%s:%s", dockerSrcURL, version)
 }
+
+func (n *Node) GetHeight(executor *knuu.Executor) (int64, error) {
+	status, err := getStatus(executor, n.Instance)
+	if err == nil {
+		blockHeight, err := latestBlockHeightFromStatus(status)
+		if err == nil {
+			return blockHeight, nil
+		}
+	}
+	if errors.Is(err, context.Canceled) || errors.Is(err, context.DeadlineExceeded) {
+		return 0, err
+	}
+
+	return 0, fmt.Errorf("error getting height: %w", err)
```

Comment:
[nit]
What's the difference between the two returns? one is adding the string `"error getting height:"`, but does it matter? is it being parsed somewhere?